src/utils/computeNextRun.ts

```diff
@@ -40,10 +46,27 @@ export default function computeNextRun(
   // If there is an interval and this is not the first run, just compute the
```

Comment:
for the `!previousStart` case above, do we want to do the `isValidNextRun()` check as well? if for instance i set a schedule end time that's some time in the past, should the first run happen? i'd be inclined to say it shouldn't

Reply:
Yeah I can change the behavior but the current behavior was that nextrunTime can be set in the past and the job will be executed as soon as possible. I can add a check there to change that behavior so the nextRunTime is always valid. Will fix that